test/python/unit/api/test_term.py

```diff
@@ -0,0 +1,789 @@
+###############################################################################
+# Top contributors (to current version):
+#   Makai Mann, Andres Noetzli
+#
+# This file is part of the cvc5 project.
+#
+# Copyright (c) 2009-2021 by the authors listed in the file AUTHORS
+# in the top-level source directory and their institutional affiliations.
+# All rights reserved.  See the file COPYING in the top-level source
+# directory for licensing information.
+# #############################################################################
+##
+
+
+
+import pytest
+import pycvc5
+from pycvc5 import kinds
+from pycvc5 import Sort, Term
+
+def assert_eq(x, y):
+    assert x == y
+
+@pytest.fixture
+def solver():
+    return pycvc5.Solver()
+
+
+def test_eq(solver):
+  uSort = solver.mkUninterpretedSort("u")
+  x = solver.mkVar(uSort, "x")
+  y = solver.mkVar(uSort, "y")
+  z = Term(solver)
+
+  assert x == x
+  assert not x != x
+  assert not x == y
+  assert x != y
+  assert not (x == z)
+  assert x != z
+
+def test_getKind(solver):
```

Comment:
The other new tests all used underscores and lowercase (https://github.com/CVC4/CVC4/blob/8431e9d49b71774092ca29c85855cbdf5bf09c53/test/python/unit/api/test_sort.py#L64), but these use a combination of underscores and camel case. Either is fine from my point of view, but they should probably be the same.